docker-compose/op-scim.service

```diff
@@ -0,0 +1,17 @@
+#/etc/systemd/system/op-scim.service 
```

Comment:
I would change path to systemd unit file to `/lib/systemd/system/op-scim.service`

Reply:
this would depend on the distribution i imagine. on the CentOS VM that I'm playing with it's stored where I have it commented.